src/ansys/fluent/core/services/datamodel_se.py

```diff
@@ -1443,6 +1460,13 @@ def _get_file_purpose(self, arg):
             elif file_purpose == "output":
                 if _OutputFile not in self.__class__.__bases__:
                     self.__class__.__bases__ += (_OutputFile,)
+            elif file_purpose == "inout":
+                if _InOutFile not in self.__class__.__bases__:
+                    self.__class__.__bases__ += (_InOutFile,)
+            else:
```

Comment:
check if file_purpose is not None outside.

Reply:
Done. Thank you.